### PR TITLE
[WIP Docs] Updated M3 Docs Format and Content

### PR DESCRIPTION
**Pull request by gibbscullen**
**What this PR does / why we need it**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing and/or backwards incompatible change?**:

```release-note

```

**Does this PR require updating code package or user-facing documentation?**:

```documentation-note

```

### CODE REVIEW

**Comment by benraskin92** [comment]
so I think all docs need to have a specific header: e.g.
```
---
title: "Grafana"
date: 2020-03-02T14:11:31-05:00
# draft: true
---
```

**Comment by gibbscullen** [comment]
image

File: docs-beta/content/getting_started/kube.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+
+You can now proceed to initialize a namespace and placement for the cluster the same as you would for our other how-to guides:
+#### Open a local connection to the coordinator service:
+$ kubectl -n m3db port-forward svc/m3coordinator 7201
+Forwarding from 127.0.0.1:7201 -> 7201
+Forwarding from [::1]:7201 -> 7201
+
+#### Create an initial cluster topology
+curl -sSf -X POST localhost:7201/api/v1/placement/init -d '{
+    "num_shards": 1024,
```
Comment (xcode03) [inline docs-beta/content/getting_started/kube.md:60]: I think the number of shards here should be 64?

Comment (gibbscullen) [reply docs-beta/content/getting_started/kube.md:60]: @xcode03 - thanks! We'll take a look.

**Pull request closed without merging**